applications/StructuralMechanicsApplication/custom_constitutive/elastic_isotropic_3d.cpp

```diff
@@ -246,7 +249,10 @@ Vector& ElasticIsotropic3D::CalculateValue(
         r_flags.Set( ConstitutiveLaw::COMPUTE_STRESS, true );
 
         this->CalculateMaterialResponsePK2(rParameterValues);
-        rValue = rParameterValues.GetStressVector();
+        if (rValue.size() != GetStrainSize()) {
```

Comment:
Interesting that any of us detected this before.. T.T

Reply:
We should sue blame...